java/java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/LinuxJavaPlatformDetector.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.java.j2seplatform;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ *
+ * @author lkishalmi
+ */
+public class LinuxJavaPlatformDetector implements Runnable {
```

Comment:
I like the work here but I would change the class name to `UnixPlatformDetector`..

There are platforms out there that are definitely not Linux but act  the same as Linux (i..e. it's just another Unix platform). Also, there are platforms that use the same filesystem layout but are also not Linux.

Reply:
yeah thought about it but I don't want to rename it until the pattern is actually matching on other distributions. It is all internal code so we can rename it any time we want. Its tested on debian, fedora and manjaro which is all linux.

Reply:
Understand.

Reply:
I rename it if you test it on unix :) This workflow created a dev-build

Reply:
I could make that happen..